### PR TITLE
feat: integrate L1 mock with services

### PR DESCRIPTION
**Pull request by mrekucci**
## Describe your changes

L1 mock integration with all services within the cluster

Fixes # (issue)

## Checklist before requesting a review

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: contracts/entrypoint.sh [hunk 1]
```diff
@@ -31,10 +31,39 @@ if [ "${DEPLOY_TYPE}" = "core" ]; then
...
-        --force \
         --json \
         --via-ir
-fi 
+elif [ "${DEPLOY_TYPE}" = "settlement-gateway" ]; then
+    echo "Deploying gateway contract on settlement chain"
+    "${FORGE_BIN_PATH}" script \
+        "${SCRIPT_PATH_PREFIX}/standard-bridge/DeployStandardBridge.s.sol:DeploySettlementGateway" \
+        --root "${CONTRACT_REPO_ROOT_PATH}" \
+        --priority-gas-price 2000000000 \
```
Comment (shaspitz) [inline contracts/entrypoint.sh:43]: Not a big deal but I'd bet the newer versions of foundry don't need explicit gas params to be set anymore

File: infrastructure/nomad/playbooks/templates/jobs/contracts-deployer.nomad.j2 [hunk 2]
```diff
@@ -92,19 +97,16 @@ job "{{ job.name }}" {
 
       template {
         data = <<-EOH
+          L1_FINALIZATION_FEE="1"
```
Comment (shaspitz) [inline infrastructure/nomad/playbooks/templates/jobs/contracts-deployer.nomad.j2:100]: Good for now (mock env) but we'll need to change these for prod

**Review by shaspitz — COMMENTED** [review]
Nice!

File: bridge/standard/pkg/node/node.go [hunk 3]
```diff
@@ -0,0 +1,401 @@
+package node
```
[the remark this answers is not in the document]
Comment (aloknerurkar) [reply bridge/standard/pkg/node/node.go:1]: Yes

File: bridge/README.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ This repository houses a purpose-built lock/mint bridge implementation, includin
```
[the remark this answers is not in the document]
Comment (mrekucci) [reply bridge/README.md:4]: Done.

File: bridge/standard/pkg/node/node.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (mrekucci) [reply bridge/standard/pkg/node/node.go:1]: Yes.

File: bridge/standard/pkg/relayer/relayer.go [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+						"error in settlement finalization",
+						"recipient", upd.Recipient,
+						"amount", upd.Amount,
+						"transferIdx", upd.TransferIdx,
+						"error", err,
+					)
+					r.metrics.failedFinalizations.WithLabelValues("settlement").Inc()
+					continue
+				}
+				r.metrics.finalizedTransfers.WithLabelValues("settlement").Inc()
```
[the remark this answers is not in the document]
Comment (mrekucci) [reply bridge/standard/pkg/relayer/relayer.go:87]: @aloknerurkar

File: contracts/entrypoint.sh (hunk 1 above)
Comment (mrekucci) [reply contracts/entrypoint.sh:43]: Unfortunately, without it, the contract deployment hangs forever.

**Review by aloknerurkar — APPROVED** [review]

**Pull request merged**